R/structural.properties.R

```diff
@@ -641,6 +645,10 @@ degree <- function(graph, v = V(graph),
   res
 }
 
+#' @rdname degree
```

Comment:
there is no specific rd for max_degree() because you chose to document on the same page as `degree()`